### PR TITLE
fix flake with OBC test suite

### PR DESCRIPTION
**Pull request by SanjalKatiyar**

### CODE REVIEW

File: cypress/views/obcPage.ts [hunk 1]
```diff
@@ -17,19 +17,21 @@ export class CreateOBCHandler {
...
-    cy.byTestID('item-create').click();
-    cy.byTestID('obc-name').type(this.name);
-    cy.byTestID('loading-indicator').should('not.exist');
-    cy.byTestID('sc-dropdown').should('be.visible').click();
-    cy.contains('openshift-storage.noobaa.io').click();
-    cy.byTestID('obc-create').click();
-    cy.byLegacyTestID('resource-title').contains(this.name, {
-      timeout: MINUTE,
+    cy.get('#page-sidebar').contains('Data Foundation'); // eslint-disable-line cypress/require-data-selectors
+    cy.clickNavLink(['Storage', 'Object Bucket Claims']).then(() => {
```
Comment (bipuladh) [inline cypress/views/obcPage.ts:21]: is `then` required ?

Comment (SanjalKatiyar) [reply cypress/views/obcPage.ts:21]: not really...
but I added it anyway, seems alright to me... but I can remove that if u suggest ??

Comment (bipuladh) [reply cypress/views/obcPage.ts:21]: lets remove it.

Comment (SanjalKatiyar) [reply cypress/views/obcPage.ts:21]: okay I will remove it...
just to confirm, can there be any issue with using then ??

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**